core/src/builder/transaction/operator_collateral.rs

```diff
@@ -184,11 +192,15 @@ pub fn create_kickoff_timeout_txhandler(
 ) -> Result<TxHandler, BridgeError> {
     let builder = TxHandlerBuilder::new()
         .add_input(
+            NormalSignatureKind::KickoffTimeout1,
             kickoff_tx_handler.get_spendable_output(3)?,
+            SpendPath::ScriptSpend(1),
```

Comment:
What's that 1? Magic number?

Reply:
yeah, we have a script indexing problem. We refer to scripts by their index. We should be tagging them someway else, open to suggestions here.